src/knot/dnssec/zone-events.h

```diff
@@ -36,17 +36,17 @@ typedef enum {
 	KEY_ROLL_FORCE_KSK_ROLL    = (1 << 1),
 	KEY_ROLL_ALLOW_ZSK_ROLL    = (1 << 2),
 	KEY_ROLL_FORCE_ZSK_ROLL    = (1 << 3),
-	KEY_ROLL_ALLOW_NSEC3RESALT = (1 << 4),
+	KEY_ROLL_ALLOW_NSEC3RE_SALT = (1 << 4),
```

Comment:
This broke the indentation of the RHS. If the change is acceptable, the indentation should be fixed. If it isn't, then I've saved myself the work of changing all the other lines :-)